src/test/java/org/owasp/wrongsecrets/challenges/kubernetes/Challenge46Test.java

```diff
@@ -0,0 +1,34 @@
+package org.owasp.wrongsecrets.challenges.kubernetes;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class Challenge46Test {
+
```

Comment:
not sure if we can have the injector as part of a test, @nbaars would you know?

Reply:
This only creates a instance of the config class. Looks good to me.